src/CalcViewModel/HistoryViewModel.cpp

```diff
@@ -129,6 +131,11 @@ void HistoryViewModel::DeleteItem(_In_ HistoryItemViewModel ^ e)
             RaisePropertyChanged(ItemsSizeKey);
         }
     }
+    wstring localizedIndex = to_wstring(itemIndex + 1);
```

Comment:
Can you add a comment prefixing this line on why we're using itemIndex + 1?

Reply:
Added a small comment to indicate the motivation behind the change.  Essentially, I wanted to move away from zero based numbering to 1-based so it would provide a better experience for end users.   Please let me know if this is in the right direction.

Reply:
Great! This makes total sense.